__tests__/app.test.js

```diff
@@ -312,4 +312,63 @@ describe("GET: /api/reviews/:reviews_id/comments", () => {
             expect(body.msg).toBe("page not found");
         })
     })
-});
\ No newline at end of file
+});
+
+//A test battery that creates a new comment on a review_id endpoint, with the correct properties
+describe("POST: /api/review/:review_id/comments", () => {
+    test("status 201: posts a new comment with the appropriate properties", () => {
+        const newComment = {
+            username: "mallionaire",
+            body: "maaa maaaa.... pwwwhhh.... ba!"
+        }
+        return request(app)
+        .post("/api/reviews/3/comments")
+        .send(newComment)
+        .expect(201)
+        .then (({ body}) => {
+            expect(body.comment).toEqual(expect.objectContaining({
+                review_id: 3,
+                author: "mallionaire",
+                body: "maaa maaaa.... pwwwhhh.... ba!"
+            }));
+        });
+    });
+    test("Status 400: body does not contain both mandatory keys", () => {
+        const newComment = {
+            username: "mallionaire"
+        }
+        return request(app)
+        .post("/api/reviews/3/comments")
+        .send(newComment)
+        .expect(400)
+        .then (({ body }) => {
+            expect(body.msg).toBe("bad request");
```

Comment:
Youre only testing to see if there is no message, but there is no test for if the username key is not included